Documentation/network/clustermesh/aks-clustermesh-prep.rst

```diff
@@ -67,6 +67,9 @@ Install cluster one
     except for Azure reserved ranges and ranges used by other clusters we intend to
     add to the clustermesh.
 
+    .. note::
+        We need to set explicit values for pod-cidr and service-cidr else AKS will allocate (``10.244.0.0/16``) for pod-cidr and (``10.0.0.0/16``) for service-cidr.
+
```

Comment:
```suggestion
    .. note::
        We need to set explicit values for pod CIDR and service CIDR, otherwise AKS will
        allocate ``10.244.0.0/16`` for pod CIDR and ``10.0.0.0/16`` for service CIDR.

```